src/sgame/components/TurretComponent.cpp

```diff
@@ -1,19 +1,19 @@
 #include "TurretComponent.h"
 #include "../Entities.h"
```

Comment:
This file is in the "new style" as described at https://wiki.unvanquished.net/wiki/Coding_convention. Please remove the `for readability's sake!` commit.

Reply:
I'm OK to look away from this if it makes easier to merge this PR. I know how it can be hard to rebase some work _after_ such change. This can be a mistake but I prefer time and effort invested on making the game better. Or maybe freem can just revert the style as an ending commit, without rebasing everything.

We have to reduce contribution cost and review cost.